paleontographica-abteilung-b-palaeobotany-palaeophytology.csl

```diff
@@ -0,0 +1,246 @@
+          <text variable="container-title"/>
+          <text macro="edition"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="any">
+        <choose>
+          <if is-numeric="edition">
+            <number variable="edition" form="long-ordinal" text-case="capitalize-first"/>
+            <text term="edition" form="long" prefix=" " text-case="capitalize-first"/>
+          </if>
+          <else>
+            <text variable="edition"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="number-of-pages">
+    <choose>
+      <if type="book report" match="any">
+        <group delimiter=" ">
+          <label plural="always" strip-periods="false" variable="page" form="short"/>
+          <text variable="number-of-pages"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <text variable="container-title" font-style="normal"/>
+        <text variable="volume" font-weight="normal" prefix=", "/>
+        <text variable="issue" prefix=" (" suffix=")"/>
+        <text variable="page" prefix=": "/>
+      </if>
+      <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="any">
+        <group>
+          <text term="in" suffix=": " text-case="capitalize-first"/>
+          <text macro="container-contributors" suffix=": "/>
+          <text macro="container-title" suffix="."/>
+          <label variable="page" form="short" suffix=". " prefix=" – "/>
+          <text variable="page"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" collapse="year" disambiguate-add-givenname="true" givenname-disambiguation-rule="by-cite">
+    <sort>
+      <key variable="issued"/>
+      <key macro="author"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=": ">
+        <group delimiter=" ">
+          <text macro="author-short"/>
+          <text macro="issued-year"/>
+        </group>
+        <text macro="citation-locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" entry-spacing="1" line-spacing="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout suffix=".">
+      <text macro="author"/>
+      <text macro="issued" prefix=" (" suffix="): "/>
+      <text macro="title" suffix=". – "/>
+      <group delimiter=". " suffix=" ">
```

Comment:
don't use groups with a single element. They don't do anything. Move the element out and apply the suffix to it directly.